connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -283,9 +283,12 @@ object SparkConnectService {
     startGRPCService()
   }
 
-  def stop(): Unit = {
+  def stop(timeout: Option[Long] = None, unit: Option[TimeUnit] = None): Unit = {
     if (server != null) {
       server.shutdownNow()
```

Comment:
I haven't studied the `shutdownNow()` api, but it doesn't seem to be a Graceful shutdown. Don't we need it?